x-pack/plugins/security/server/authentication/tokens.ts

```diff
@@ -103,8 +103,15 @@ export class Tokens {
         ).invalidated_tokens;
       } catch (err) {
         this.logger.debug(`Failed to invalidate refresh token: ${err.message}`);
-        // We don't re-throw the error here to have a chance to invalidate access token if it's provided.
-        invalidationError = err;
+
```

Comment:
At some point (not now), it might make sense to consolidate the error handling logic for the access and refresh tokens, since they appear nearly identical. I fear these will accidentally diverge at some point, and we'll have another subtle auth bug on our hands

Reply:
Totally agree, every time I change this code I stare at it and want to unify, but always defer for one reason or another :see_no_evil: